engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentUtils.scala

```diff
@@ -1,21 +1,18 @@
 package pl.touk.nussknacker.k8s.manager.deployment
 
 import com.typesafe.config.{Config, ConfigFactory, ConfigRenderOptions}
-import org.apache.commons.io.IOUtils
 import play.api.libs.json.Json
 import skuber.apps.v1.Deployment
 
 object DeploymentUtils {
 
-  def createDeployment(deploymentString: String): Deployment = {
-    val deploymentSpec = Json.parse(deploymentString).as[Deployment]
-    deploymentSpec
-  }
-
-  def createDeployment(config: Config): Deployment = {
-    val minimalPlaceholderConfig = ConfigFactory.parseString(Json.parse(IOUtils.toString(getClass.getResourceAsStream(s"/deploymentMinimal.json"))).toString())
-    val mergedConfig = config.withFallback(minimalPlaceholderConfig)
+  def parseDeploymentWithFallback(config: Config): Deployment = {
+    val defaultMinimalDeploymentConfig = ConfigFactory.parseURL(getClass.getResource(s"/defaultMinimalDeployment.conf"))
```

Comment:
Can you pass `getClass.getResource(s"/defaultMinimalDeployment.conf")` as a parameter to this method? It looks like service logic (`DeploymentPreparer`) because it contains default now - not general purpose utility logic.

Reply:
done